index.js

```diff
@@ -144,6 +157,38 @@ bot.onText(/^\/autodelete(@sticker_time_bot)?(\s+([^\s]+))?$/, (msg, match) => {
     }
 });
 
+bot.onText(/\/sleeptime (\d+)/, (msg, match) => {
+    const chatId = msg.chat.id;
+    // bot.sendMessage(chatId, match[0]+'  '+match[1]+'  '+match[2]+'  '+match[3])
+    if (match[1]) {
+        if (match[1] <= 60 && match[1] >= 0){
+            logger.info(chatId + ' set sleeptime to '+match[1]+':00');
+            bot.sendMessage(chatId, 'Set sleeptime to '+match[1]+':00');
+            data.sleeptime[chatId] = match[1]
+            saveData();
+        } else {
+            bot.sendMessage(chatId, 'Invalid time, 0-23 expected');
+            bot.sendSticker(chatId, stickers[13])
+        }
+    }
```

Comment:
You should give some feedback to the user instead of just ignoring invalid inputs.